freemind.json

```diff
@@ -0,0 +1,21 @@
+{
+    "homepage": "http://freemind.sourceforge.net/wiki/index.php/Main_Page",
+    "version": "1.0.1",
+    "description": "FreeMind is a free mind mapping application written in Java. It provides extensive export capabilities.",
+    "license": "GNU GPL V2+",
+    "hash": "005e815bf5d1867eadcaa89194df9952d986ace9eed0a3058c7ee6255799a60a",
+    "url": "https://netix.dl.sourceforge.net/project/freemind/freemind/1.0.1/freemind-bin-max-1.0.1.zip",
```

Comment:
Use sourceforges direct download mirror: `https://downloads.sourceforge.net/project/freemind/freemind/1.0.1/freemind-bin-max-1.0.1.zip` with hash `8b4c16f253d5c56b7fddf6a1878831556febfc48` found in https://sourceforge.net/projects/freemind/files/freemind/1.0.1/

Reply:
Oops, merged.